packages/@glimmer/wire-format/lib/elements.ts

```diff
@@ -0,0 +1,60 @@
+export enum Tags {
```

Comment:
Would it worth to add svg tags here?
Future idea: Is it possible to statically gather the used tags analysing the templates?